functions/src/index.ts

```diff
@@ -17,3 +20,32 @@ import * as logger from "firebase-functions/logger";
 //   logger.info("Hello logs!", {structuredData: true});
 //   response.send("Hello from Firebase!");
 // });
+export const addReviewerRole = functions.https.onCall((data: any, context: any) => {
+    return admin.auth().getUserByEmail(data.email).then((user: any) => {
+        return admin.auth().setCustomUserClaims(user.uid, {
+            role: "reviewer"
+        });
+    }
+    ).then(() => {
+        return {
+            message: `Success! ${data.email} has been made an admin.`
```

Comment:
Small thing, this returns "has been made an admin" for any user role. It should say "reviewer" for reviewer etc.